.gitignore

```diff
@@ -1,2 +1,3 @@
 plugins/
 vendor/
+.idea
```

Comment:
`.idea` should be added at a developer's global level (`~/.gitignore`) not in every plugin's `.gitignore` IMO.